### ISSUE TITLE
Prevent keyboard trap in lesson editor

### ISSUE THREAD

**Issue by jrobind** [jrobind/dev-jot#2]
Currently if you tab into the editor you cannot tab out of it. This causes a keyboard trap which is bad accessibility. 

Essentially, we need to prevent [WCAG 2.1.2 No Keyboard Trap.
](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-trapping.html)

A possible solution is to allow the escape key to exit the editor.
 

After some discussion in the Discord channel we might want to approach this by:

* adding a checkbox that allows you to enable/disable tab inside the quill editor. Once active you'll need to press esc (or click) to deactivate tab allowed mode
* alongside a checkbox, we might also need an onscreen message informing the user that hitting escape will allow them to tab to the next element.
